src/util-hugepages.c

```diff
@@ -57,7 +57,7 @@ static uint16_t SystemNodeCountGetLinux(void)
     char dir_path[] = "/sys/devices/system/node/";
```

Comment:
For example, my Raspberry PI (more or less default) does not have `/sys/devices/system/node`.  So would a Linux specific check to see if huge pages are enabled be "if linux and /sys/devices/system/node exists"?

Reply:
ok, thanks, that's a good catch, I forgot that the path might not be present on every Linux, I'll redo the path check.